client/src/components/DisplayTable/DisplayTable.tsx

```diff
@@ -76,6 +82,7 @@ const get_column_config_defaults = (index: number) => ({
   is_sortable: true,
   is_summable: false,
   is_searchable: false,
+  show_dropdown_filter: true,
```

Comment:
Left it in as a feature, not currently used. We could want it searchable but not have dropdown filter🤷‍♂️

Reply:
I'd rather remove it. 

From the dev side, this component already has a bloated API, and this is a tricky one because it's meaningless unless is_searchable` is also true. 

From the user side, I don't want the inconsistency of sometimes having the filter button and sometimes not, even if it's just in one or two obscure cases. I can't think of a reason we'd disable it, which probably means if we ever have one it'll be something really obscure in which case the user probably would have no chance to work out our hidden logic for why the filter is missing. They'd just see it as a bug.